src/templates/layouts/healthCareLocalFacility/index.tsx

```diff
@@ -1,13 +1,129 @@
-type HealthCareLocalFacilityProps = {
-  title: string
-}
+import { HealthCareLocalFacility as FormattedHealthCareLocalFacility } from '@/types/formatted/healthCareLocalFacility'
 
 export function HealthCareLocalFacility({
   title,
-}: HealthCareLocalFacilityProps) {
+  introText,
+  operatingStatusFacility,
+}: FormattedHealthCareLocalFacility) {
   return (
-    <div>
-      <p>{title}</p>
+    <div className="interior" id="content">
+      <main className="va-l-detail-page va-facility-page">
+        <div className="usa-grid usa-grid-full">
+          <div>TODO: Sidebar nav</div>
+          <div className="usa-width-three-fourths">
+            <article className="usa-content va-l-facility-detail">
+              <div>TODO: Lovell switch link</div>
+
+              {title && <h1>{title}</h1>}
+
+              {introText && (
+                <div className="va-introtext">
+                  <p>{introText}</p>
+                </div>
+              )}
+
+              <div className="usa-grid usa-grid-full vads-u-margin-y--1p5 vads-u-margin-bottom--6">
+                <div>TODO: facilities_health_services_buttons</div>
+              </div>
+
+              <va-on-this-page></va-on-this-page>
+
+              {/* Main content */}
+              <h2 className="vads-u-line-height--1 vads-u-margin-bottom--3">
+                Location and contact information
+              </h2>
+              <div className="region-list usa-grid usa-grid-full vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row facility vads-u-margin-bottom--2p5 vads-u-margin-bottom--4">
+                <div className="usa-width-two-thirds vads-u-display--block vads-u-width--full">
+                  <div>
+                    <div className="vads-c-facility-detail">
+                      <div>TODO: Operating status flags</div>
+                      <section className="vads-facility-detail">
+                        <script type="application/ld+json">
+                          {/* TODO: Fill this in */}
+                        </script>
```

Comment:
Not sure how this will work yet. 🤷 I've never added a `<script>` element inside React before. 😅